_redirects.yaml

```diff
@@ -456,7 +474,7 @@ redirects:
 - from: /tutorials/webgl/typed_arrays/
   to: https://web.dev/webgl-typed-arrays
 
-- from: /webgl/webgl_fundamentals/
+- from: tutorials/webgl/webgl_fundamentals/
```

Comment:
Should there be a / ? the other tutorials have them

Reply:
good catch, fixed.